app/assets/stylesheets/application.scss

```diff
@@ -133,6 +133,10 @@ body {
   }
 }
 
+.main-header .toolbar__item {
+  color: white;
+}
+
```

Comment:
fyi, this is for the language switcher link that is commented out until we turn spanish on in production. i think we added some new header links for zendesk admin while i was working on this too but i couldn't log in at /zednesk/sign-in in dev to make sure it plays nice with the header changes i made.

Reply:
Good to know, thanks.